### DISCUSSION TITLE
Discord Nitro (2x boost) giveaway.

### DISCUSSION THREAD

**Discussion by vLuckyyy** [EternalCodeTeam/EternalCore#780] [Announcements]
The giveaway has been announced on our official discord https://discord.gg/eternalcode or https://discord.gg/FQ7jmGBd6c. To fulfill the conditions, you must leave your discord nickname in the thread below and fulfill the rest of the conditions described on the discord.

**Comment by igoyek** [comment]
igoyek, i hope i win 😄

**Comment by WieszczY85** [comment]
WieszczY 👍

**Comment by vbagieta** [comment]
vbagieta

**Comment by eripe14** [comment]
eripe14

**Comment by kamipl818** [comment]
kamipl

**Comment by seakxd** [comment]
jonathanbrick

**Comment by Hyd3r1** [comment]
khallercom

**Comment by bebeczka2** [comment]
bebeczka

**Comment by lapinfache** [comment]
lapinfache

**Comment by kamilguszpit** [comment]
kequer

**Comment by Rollczi** [comment]
Siema

**Comment by miko8181** [comment]
miko1818
